pandas/core/frame.py

```diff
@@ -1450,11 +1450,21 @@ def to_stata(self, fname, convert_dates=None, write_index=True,
             format that you want to use for the dates. Options are
             'tc', 'td', 'tm', 'tw', 'th', 'tq', 'ty'. Column can be either a
             number or a name.
+        write_index : bool
+            Write the index to Stata dataset.
         encoding : str
             Default is latin-1. Note that Stata does not support unicode.
         byteorder : str
             Can be ">", "<", "little", or "big". The default is None which uses
             `sys.byteorder`
+        time_stamp : datetime
+            A date time to use when writing the file.  Can be None, in which
+            case the current time is used.
+        dataset_label : str
+            A label for the data set.  Should be 80 characters or smaller.
+        variable_labels : dict
```

Comment:
Maybe add a note here about the variables needing to be latin-1 encodable?  Also perhaps note that a ValueError is raised if not, or if any label is too long.